rclrs/src/lib.rs

```diff
@@ -189,7 +193,7 @@ mod service;
 mod subscription;
 mod time;
 mod time_source;
-mod vendor;
+pub mod vendor;
```

Comment:
Oh this needs to be pub because doctest [only links to public stuff](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#include-items-only-when-collecting-doctests)... Jeez, I can see this causing some confusion. I think we need to re-evaluate our message consumption more and more everyday :sweat_smile:

Reply:
Yeah, and I had to modify the vendorize script to not complain about missing docs for messages because now they are public 😅